peer/main_test.go

```diff
@@ -15,12 +15,34 @@ import (
 	"testing"
 	"time"
 
+	"github.com/hyperledger/fabric/integration/runner"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/gbytes"
 	"github.com/onsi/gomega/gexec"
 )
 
+func couchDBSetup() (addr string, cleanup func()) {
```

Comment:
Can this be replaced with the SetupExtTestEnv?

Reply:
I remove it